security_options.rst

```diff
@@ -167,32 +238,30 @@ file and enter the container with full capabilities using the
    1 packets transmitted, 1 received, 0% packet loss, time 0ms
    rtt min/avg/max/mdev = 18.838/18.838/18.838/0.000 ms
 
-``--drop-caps``
-===============
+In the OCI-mode, the ``--keep-privs`` option can be used by any user. In this
+mode, ``--keep-privs`` will cause the container to run inheriting the current
+effective capabilities rather than using the OCI default capability set. When
+entering the container as a non-root user, the capabilities are only inherited
+to the bounding set.
 
-By default, the root user has a full set of capabilities when they enter
-the container. You may choose to drop specific capabilities when you
-initiate a container as root to enhance security.
-
-For instance, to drop the ability for the root user to open a raw socket
-inside the container:
-
-.. code::
+``--no-privs``
+==============
 
-   $ sudo singularity exec --drop-caps CAP_NET_RAW library://centos ping -c 1 8.8.8.8
-   ping: socket: Operation not permitted
+In the default non-OCI-mode, the ``--no-privs`` flag allows the root user to run
+a container with all capabilities dropped, and sets the ``no_new_privs`` bit
+that will prevent the container process gaining any further privilege.
 
-The ``drop-caps`` option will also accept the case insensitive keyword
-``all`` as an option to drop all capabilities when entering the
-container.
+In OCI-mode, the ``--no-privs`` can be used by any user to run a container with
+all capabilities dropped, and sets the ``no_new_privs`` bit that will prevent
+the container process gaining any further privilege.
 
 ``--security``
 ==============
 
-The ``--security`` flag allows the root user to leverage security
-modules such as SELinux, AppArmor, and seccomp within your {Singularity}
-container. You can also change the UID and GID of the user within the
-container at runtime.
+The ``--security`` flag, currently support in non-OCI-mode only, allows the
+root user to leverage security modules such as SELinux, AppArmor, and seccomp
+within your {Singularity} container. You can also change the UID and GID of the
```

Comment:
Nit: I found the switch from 3rd person ("... allows the root user ...") to 2nd person ("... within your {Singularity} container ...") a little awkward. If possible, it would be good to rephrase this paragraph so it's more consistent in that respect.